src/lib/core/WeaveFabricState.cpp

```diff
@@ -113,6 +113,10 @@ void WeaveSessionKey::Init(void)
     NodeId = kNodeIdNotSpecified;
     NextMsgId.Init(0);
     MaxRcvdMsgId = 0;
+    InitialSendMsgId = 0;
+    InitialRcvdMsgId = 0;
+    ResumptionSendMsgId = 0;
```

Comment:
Should all this new code ifdefed with WEAVE_CONFIG_PERSIST_CONNECTED_SESSION ?

Reply:
In FabricState, I am reusing the Suspend and Restore functions that also do persistence for end-to-end CASE sessions over WRM. So, I only had that WEAVE_CONFIG within the Tunnel code.